### ISSUE TITLE
KDE Plasma additions for K27

### ISSUE THREAD

**Issue by DarkEra** [kororaproject/kp#95]
- Include breeze-gtk which is a Breeze widget theme for GTK2 and GTK3

**Comment by jimkp** [comment]
Add sddm-themes and sddm-theme-hawaii which both provide additional login themes

**Comment by DarkEra** [comment]
I'm not sure about including them, especially since Hawaii merged with Papyros became Liri OS, so i guess Fedora still ships the older packages.
The Other SDDM themes i'm not too fond of, i tried the Maui theme on F26 LXQt on the netbook a while ago but it was that slow i started to dislike it. According to the github page the themes included are sample themes so other can use it as starting point for new themes.

I'll have a look but still am in doubt....

**Comment by DarkEra** [comment]
Ok, i installed the sddm-themes package and it only includes two themes: Elarun (which looks like the KDE4 login screen from back then) and Maldives which give me a 90s feel.
Not impressive enough to include them from my point of view.

**Comment by jimkp** [comment]
> Ok, i installed the sddm-themes package and it only includes two themes: Elarun (which looks like the KDE4 login screen from back then) and Maldives which give me a 90s feel.
> Not impressive enough to include them from my point of view.

I agree that none of the themes are much good, actually in my opinion the Hawaii one is the best looking. However any of them are better than the default white on white scheme which is rather depressing.
It doesn't hurt to include them and give users an option, no one is forced to use them.
